### ISSUE TITLE
Peer review

### ISSUE THREAD

**Issue by Spurrm** [spitalnyd/roughdraft#1]
[]The title of the proposal seems appropriate and exciting.
 []The abstract of the proposal provides a short summary of the proposed work.
[X] The introduction section clearly describes the proposed work (what is being proposed).
 [X]The introduction section motivates the proposed work (why do the proposed work) with the help of existing research and includes several references.
 The related work section describes relevant literature related to the proposed work.
 The related work section situates the proposed project in the broader scope of related work.
 The method section explains the process to be utilized in the proposed study (how the proposed work will be completed).
 The method section includes the description of algorithms, programming languages, platforms and software (as relevant) that will be used in the proposed project.
 The method section includes a demonstration of feasibility.
 [X]The evaluation section describes the process to be undertaken to validate the proposed work using applicable evaluation metrics.
 [X]The proposal includes a detailed schedule.
 [X]The conclusion section discusses the impact of the proposed work and outlines avenues for future work.
 The proposal includes at least twelve references.
 The proposal consists of at least ten pages.
 [X]The proposal follows a logical flow.
 The proposal includes visual aids in terms of figures, tables, and/or graphs.
 [X]There are no typographical or grammatical errors in the proposal draft.
 [X]There is no extraneous text.

**Comment by Spurrm** [comment]
Overall, the start of this proposal is pretty good. For the parts that were included, there was a logical flow. In each part, it was well explained and was clear in what was wanting to be done. The only negative aspects of this proposal that I noticed were, that not all of the parts were included, although he submitted the completed parts which makes sense. I did not see a title in this, so I'm not sure if there is one or not, or if it is a good title. Once all the parts are submitted, I believe it to be a good proposal  given that it meets all the requirements listed above.
